README.md

```diff
@@ -16,13 +16,13 @@ Documentation
 -------------
 
 Current documentation can be found
-[here](http://docs.mongodb.org/ecosystem/tutorial/ruby-bson-tutorial/#ruby-bson-tutorial).
+[here](https://www.mongodb.com/docs/ruby-driver/current/bson-tutorials/).
```

Comment:
I decided to go to the general ruby page instead of the v4 tutorial, so we don't have to change it every time a new bson-ruby version comes out.